apps/docs/content/3-components/2-library/22-link.mdx

```diff
@@ -2,13 +2,197 @@
+    </TabList>
+    <TabPanel value="tab11">        
+        ```html
+        <a href="#" class="gi-link"></a>
+        ```
+    </TabPanel>
+    <TabPanel value="tab12">        
+        ```html
+        {{ govieLink({"href": "#", "label": "Link with blue color"}) }}
+        ```
+    </TabPanel>
+    <TabPanel value="tab13">
+        ```html
+        import { Link} from '@govie-ds/react';
+
+        <Link href="#">This is a sample link</Link>
+        ```
+    </TabPanel>
+</Tabs>
+
+### Link with inherited color
+
+<Paragraph>
+    <Link href="#" noColor>Link with inherited color</Link>
+</Paragraph>
+
+<Tabs id="tab2">
+    <TabList>
+        <TabItem value="tab21" checked>HTML</TabItem>
+        <TabItem value="tab22">Macro</TabItem>
+        <TabItem value="tab23">React</TabItem>
+    </TabList>
+    <TabPanel value="tab21">        
+        ```html
+        <a href="#" class="gi-link gi-link-inherit">Link with inherited color</a>
+        ```
+    </TabPanel>
+    <TabPanel value="tab22">        
+        ```html
+        {{ 
+            govieLink({
+                "href": "#", 
+                "label": "Link with inherited color", 
+                "noColor": true
+            }) 
+        }}
+        ```
+    </TabPanel>
+    <TabPanel value="tab23">
+        ```html
+        import { Link} from '@govie-ds/react';
+
+        <Link href="#" noColor>Link with inherited color</Link>
+        ```
+    </TabPanel>
+</Tabs>
+
+### Link with no underline
+
+<Paragraph>
+    <Link href="#">Link with no underline</Link>
+</Paragraph>
+
+<Tabs id="tab3">
+    <TabList>
+        <TabItem value="tab31" checked>HTML</TabItem>
+        <TabItem value="tab32">Macro</TabItem>
+        <TabItem value="tab33">React</TabItem>
+    </TabList>
+    <TabPanel value="tab31">        
+        ```html
+        <a href="#" class="gi-link gi-link-no-underline">Link with no underline</a>
+        ```
+    </TabPanel>
+    <TabPanel value="tab32">        
+        ```html
+        {{ 
+            govieLink({
+                "href": "#", 
+                "label": "Link with no underline", 
+                "noUnderline": true
+                }) 
+        }}
+        ```
+    </TabPanel>
+    <TabPanel value="tab33">
+        ```html
+        import { Link} from '@govie-ds/react';
+
+        <Link href="#" noUnderline>Link with no underline</Link>
+        ```
+    </TabPanel>
+</Tabs>
+
+### Link with no visited color
+
+<Paragraph>
+    <Link href="#">Link with no visited color</Link>
+</Paragraph>
+
+<Tabs id="tab4">
+    <TabList>
+        <TabItem value="tab41" checked>HTML</TabItem>
+        <TabItem value="tab42">Macro</TabItem>
+        <TabItem value="tab43">React</TabItem>
+    </TabList>
+    <TabPanel value="tab41">        
+        ```html
+        <a href="#" class="gi-link gi-link-no-visited">Link with no visited color</a>
+        ```
+    </TabPanel>
+    <TabPanel value="tab42">        
+        ```html
+        {{ 
+            govieLink({
```

Comment:
The only problem I see with this approach is that we will have to consistently check and update the docs if there are any props or updates to the component.

Reply:
Not sure if there's an alternative way.

Reply:
Just thinking out loud here . What we are trying to achieve here, rendering each component with their different variants and states, doesn't storybook accomplish the same purpose ? Gives the users the ability to look at the components and the code needed to implement the component.

Reply:
It's more a showcase of the components.
The idea is to have complete documentation where the Product team, design, and developer can find all the information needed.
I believe that Storybook is a tool for a more tech target.
Also, I've created explicit classes for the HTML case. There can be users who don't have React or Nunjucks. Storybook is almost useless for them.

I agree that maintaining this can be cumbersome, but I think it is a great value as a final product.